01-atoms/buttons/Button.component.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @file Button.component.js
+ * Exports a button component.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Component that renders a button with a click handler.
+ */
+const Button = (props) => {
+  const { onClick, children, modifiers } = props;
```

Comment:
I do this both ways, but when I have a short list of props, I will often restructure them within the arguments. Just giving you an option if you like the syntax.
```
const Button = ({ onClick, children, modifiers }) => {
```